odxtools/singleecujob.py

```diff
@@ -60,6 +84,32 @@ class OutputParam:
     def __post_init__(self) -> None:
         self._dop: Optional[DopBase] = None
 
+    @staticmethod
+    def from_et(et_element, doc_frags: List[OdxDocFragment]) \
+            -> "OutputParam":
+
+        odx_id = OdxLinkId.from_et(et_element, doc_frags)
+        assert odx_id is not None
+        short_name = et_element.findtext("SHORT-NAME")
+        assert short_name is not None
+        long_name = et_element.findtext("LONG-NAME")
+        description = read_description_from_et(et_element.find("DESC"))
```

Comment:
description is always in a DESC tag, maybe move that part of the code to the common function?
```suggestion
        description = read_description_from_et(et_element)
```

Reply:
I also thought about that, but decided against it because everything else takes the element for the tag which describes the object...